yeezyCopper.js

```diff
@@ -1,6 +1,8 @@
 // EDIT THE STUFF BELOW IN ORDER TO MAKE THIS THING WORK. :)
 // I PUT NOTES ON THE RIGHT OF EACH FIELD TO HELP YOU OUT.
 
+var shipfast = true; //IF YOU WANT OVERNIGHT SHIPPING SET TO TRUE.
```

Comment:
so I would put this as a key in the `config` object. No need to have it outside and it is a "configuration" of the app.